hack/local-up-cluster.sh

```diff
@@ -131,6 +131,10 @@ sudo "${GO_OUT}/kube-scheduler" \
   --master="http://${API_HOST}:${API_PORT}" >"${SCHEDULER_LOG}" 2>&1 &
 SCHEDULER_PID=$!
 
```

Comment:
It might be better to just echo this rather than running it in case a user already has used the context name "local" for something else.